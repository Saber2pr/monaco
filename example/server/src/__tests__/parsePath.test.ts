import { parsePath } from '../'

describe('ParsePath', () => {
  it('test parse', () => {
    expect(parsePath('/path/file')).toMatchSnapshot()
  })
})
