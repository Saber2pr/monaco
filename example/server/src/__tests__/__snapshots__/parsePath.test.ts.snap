// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParsePath test parse 1`] = `
Object {
  "base": "file",
  "dir": "/path",
  "ext": "",
  "name": "file",
  "root": "/",
}
`;
